.github/workflows/builds.yml

```diff
@@ -1514,14 +1514,26 @@ jobs:
         env:
           LLVM_PROFILE_FILE: "/tmp/ut.profraw"
       - run: llvm-profdata-14 merge -o ut.profdata /tmp/ut.profraw
+      - run: ./src/suricata --list-runmodes -l /tmp
+        env:
+          LLVM_PROFILE_FILE: "/tmp/listrunmodes.profraw"
+      - run: llvm-profdata-14 merge -o listrunmodes.profdata /tmp/listrunmodes.profraw
       - run: ./src/suricata --list-keywords -l /tmp
         env:
           LLVM_PROFILE_FILE: "/tmp/lk.profraw"
-      - run: llvm-profdata-14 merge -o ut.profdata /tmp/lk.profraw
+      - run: llvm-profdata-14 merge -o lk.profdata /tmp/lk.profraw
       - run: ./src/suricata --list-app-layer-protos -l /tmp
         env:
           LLVM_PROFILE_FILE: "/tmp/la.profraw"
-      - run: llvm-profdata-14 merge -o ut.profdata /tmp/la.profraw
+      - run: llvm-profdata-14 merge -o la.profdata /tmp/la.profraw
+      - run: ./src/suricata --dump-features -c suricata.yaml -l /tmp
+        env:
+          LLVM_PROFILE_FILE: "/tmp/dumpfeatures.profraw"
+      - run: llvm-profdata-14 merge -o dumpfeatures.profdata /tmp/dumpfeatures.profraw
+      - run: ./src/suricata --dump-config -c suricata.yaml -l /tmp
+        env:
+          LLVM_PROFILE_FILE: "/tmp/dumpconfig.profraw"
```

Comment:
It matches the existing pattern, but how does the dumpfeatures profile data end up being uploaded to codecov?

Reply:
IIRC it scans for profdata files

Reply:
All I see it doing is uploaded the coverage.txt file, which I never see this one merged into. Anyways, not really a thing with this PR.